man/en/sos-report.1

```diff
@@ -362,6 +362,20 @@ be used provided all other required values (case number, upload user) are provid
 Specify a directory to upload to, if one is not specified by a vendor default location
 or if your destination server does not allow writes to '/'.
 .TP
+.B \--upload-method METHOD
+Specify the HTTP method to use for uploading to the provided --upload-url. Valid
+values are 'auto' (default), 'put', or 'post'. The use of 'auto' will default to
+the method required by the policy-default upload location, if one exists.
+
+This option has no effect on upload methods other than HTTPS.
```

Comment:
Maybe some wording problem or my understanding of English, but I read it "The --upload-method has no effect to upload methods ..". Once method = HTTP request type (over *just* HTTPS connection), once method = connection/protocol type (HTTPS/HTTP/FTP).

Maybe "upload methods" -> "upload protocols"?

Reply:
Ack.